test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,53 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+
+chai.use(chaiSubset);
+
+const urlBase = 'https://api.github.com';
+const Access = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+const query = {
```

Comment:
can you rename this as gistData? or something that tell us that this variable is storing the gist